core/src/main/scala/stryker4s/config/ConfigReader.scala

```diff
@@ -75,7 +77,7 @@ object ConfigReader {
     /** Attempt to read a config
       */
     def tryRead: Reader.Result[T] = {
-      log.info(s"Attempting to read config from stryker4s.conf")
+      log.info(s"Attempting to read config from ${defaultConfig}")
```

Comment:
This will not toString nicely... Needs to separate _file name_ from _config object_.

Reply:
Can you put the `getProperty` or default in a separate variable to properly log the file location?